File: app/src/main/java/com/example/learn/testme/ViewHolder/HistoryViewHolder.java
package com.example.learn.testme.ViewHolder;

import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.example.learn.testme.Interface.ItemClickLIstener;
import com.example.learn.testme.R;

public class HistoryViewHolder extends RecyclerView.ViewHolder {


    public TextView txt_name, txt_score, txt_time, txt_subjecteId, txt_subjecteNmae;
    private ItemClickLIstener itemClickLIstener;


    public HistoryViewHolder(@NonNull View itemView) {
        super(itemView);

        txt_name = (TextView) itemView.findViewById(R.id.his_name);
        txt_score = (TextView) itemView.findViewById(R.id.his_Score);
        txt_subjecteId = (TextView) itemView.findViewById(R.id.his_subjecte_id);
        txt_subjecteNmae = (TextView) itemView.findViewById(R.id.his_subjecte_name);
        txt_time = (TextView) itemView.findViewById(R.id.his_time);




    }


}

File: app/src/main/java/com/example/learn/testme/ViewHolder/HistoryViewHolder.kt
package com.example.learn.testme.ViewHolder

import androidx.recyclerview.widget.RecyclerView.ViewHolder
import android.view.View
import android.widget.TextView
import com.example.learn.testme.Interface.ItemClickLIstener
import com.example.learn.testme.R

class HistoryViewHolder(itemView: View) : ViewHolder(itemView) {
    var txt_name: TextView
    var txt_score: TextView
    var txt_time: TextView
    var txt_subjecteId: TextView
    var txt_subjecteNmae: TextView
    private val itemClickLIstener: ItemClickLIstener? = null

    init {
        txt_name = itemView.findViewById<View>(R.id.his_name) as TextView
        txt_score = itemView.findViewById<View>(R.id.his_Score) as TextView
        txt_subjecteId = itemView.findViewById<View>(R.id.his_subjecte_id) as TextView
        txt_subjecteNmae = itemView.findViewById<View>(R.id.his_subjecte_name) as TextView
        txt_time = itemView.findViewById<View>(R.id.his_time) as TextView
    }
}